data/web/inc/functions.inc.php

```diff
@@ -1183,19 +1183,8 @@ function get_logs($container, $lines = false) {
   }
   return false;
 }
-function getGUID() {
-  if (function_exists('com_create_guid')) {
-    return com_create_guid();
-  }
-
-  mt_srand((double)microtime()*10000);//optional for php 4.2.0 and up.
-  $charid = strtoupper(md5(uniqid(rand(), true)));
-  $hyphen = chr(45);// "-"
-
-  return substr($charid, 0, 8).$hyphen
-        .substr($charid, 8, 4).$hyphen
-        .substr($charid,12, 4).$hyphen
-        .substr($charid,16, 4).$hyphen
-        .substr($charid,20,12);
+function getGUID($id) {

```

Comment:
To prevent adding multiple configs for the same identity.

Reply:
ok

Reply:
but it should be not GUID named

Reply:
I think, that it is unnecessary change. According to docs `A globally unique identifier for the payload. The actual content is unimportant, but it must be globally unique. In macOS, you can use uuidgen to generate reasonable UUIDs.`

Reply:
Using the same UID overrides profiles, doesn't it?

Reply:
you have duplicates in one profile now

Reply:
I personaly not tested replacing profiles, but if yes, them should be only on line 160 and not on all UUIDs

Reply:
I know, that if profile name is same, it not allow to install it.

Reply:
I tested it on iPad, and my version with GUID works fine

Reply:
I tested it now, it it don't allow to install same profile twice, no matter if guid si same or not. It just checks `PayloadIdentifier`

Reply:
So I propose to revert that changes and keep for ex. just that `com_create_guid` function

Reply:
Please respond in a single comment.....

Reply:
Okay, I see. So a random GUID is not a problem, but the PayloadIdentifier needs to be different for each profile. I have seen an issue which erroneously made me assume it was the GUID, sorry.

Can you revert my changes in your pr?